src/components/app/data/hooks/useEnterpriseLearner.test.jsx

```diff
@@ -19,37 +20,94 @@ const mockEnterpriseCustomer = enterpriseCustomerFactory();
 const mockAuthenticatedUser = authenticatedUserFactory();
 const mockEnterpriseLearnerData = {
   enterpriseCustomer: mockEnterpriseCustomer,
-  enterpriseCustomerUserRoleAssignments: [],
   activeEnterpriseCustomer: null,
   activeEnterpriseCustomerUserRoleAssignments: [],
   allLinkedEnterpriseCustomerUsers: [],
   enterpriseFeatures: {},
   staffEnterpriseCustomer: null,
+  shouldUpdateActiveEnterpriseCustomerUser: false,
 };
 
+const mockBFFDashboardData = {
+  enterpriseCustomer: {
+    ...mockEnterpriseCustomer,
+    isBFFEnabled: true,
+  },
+  allLinkedEnterpriseCustomerUsers: [],
+  enterpriseFeatures: {
+    isBFFEnabled: true,
```

Comment:
nit: similar to other related comments, worth removing `isBFFEnabled` from `enterpriseFeatures` here?

Reply:
See https://github.com/openedx/frontend-app-learner-portal-enterprise/pull/1263#discussion_r1970365983